packages/react/rollup.config.js

```diff
@@ -107,6 +107,7 @@ export default [
       'components/SelectionGroup/index': 'src/components/selectionGroup/index.ts',
       'components/SideNavigation/index': 'src/components/sideNavigation/index.ts',
       'components/StatusLabel/index': 'src/components/statusLabel/index.ts',
+      'components/Table/index': 'src/components/table/index.ts',
```

Comment:
If we choose to have the separate DataTable component as well, then we probably need to add it separately to rollup as well.

Reply:
I think it might be the easiest way to start with just one table component. We could add composition support later on if someone requests it.

Reply:
I agree that would be the easiest way to start now. What do you think @laurakarhu ?

Reply:
Sounds like a good plan :)

Reply:
- [ ] Expose only Table component, rename DataTable -> Table